Samples.Swift/Samples.Swift/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIViewController+Extensions.swift
+//  paypalcheckout-iOS-swift-sample
+//
+//  Created by Jax DesMarais-Leder on 3/8/21.
+// Copyright © 2021 PayPal. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+
+  func textField(
+    placeholder: String? = nil,
+    defaultValue: String? = nil,
+    clearButton: UITextField.ViewMode = .never
+  ) -> UITextField {
+    let result = UITextField()
+    result.translatesAutoresizingMaskIntoConstraints = false
+    result.borderStyle = .roundedRect
+    result.placeholder = placeholder
+    result.text = defaultValue
+    result.clearButtonMode = clearButton
+    if let textDelegate = self as? UITextFieldDelegate {
+      result.delegate = textDelegate
+    }
+    return result
```

Comment:
Why `result`? i think this might be clearer if we just named it `textField`. What do you think ?

Reply:
Yep, I think that's more clear. This was swiped from STA and that is what was used there, but agree `textField` is clearer. 😉